sickbeard/providers/libertalia.py

```diff
@@ -99,7 +99,7 @@ def _doSearch(self, search_params, search_mode='eponly', epcount=0, age=0, epObj
                 with BS4Parser(data, features=["html5lib", "permissive"]) as html:
                     resultsTable = html.find("table", {"class" : "torrent_table"})
                     if resultsTable:
-                        rows = resultsTable.findAll("tr", {"class" : re.compile("torrent_row(.*)?")})
+                        rows = resultsTable.findAll("tr", {"class" : re.compile("torrent_row\.*")})
```

Comment:
`re.compile("torrent_row.*")` maybe, I dont have an account so I cant test it. The old solution didn't work for anything older than a few days.